components/unzipped/how-it-works-freelancer/how-it-works-content/how-it-works-content.js

```diff
@@ -0,0 +1,165 @@
+        imgSource: "/img/GetPaid.png"
+    }
+];
+
+
+const labelsSMArray = [
+    {
+        title: "Create your profile (it’s free)",
+        description: "An eye-catching title and client-focused overview help us match you to the work you want. Include your work history. Your skills, and your photo. Add more, like an introduction video to create a profile that really stands out.",
+        imgSource: "/img/BuildYourProfile.png"
+    },
+    {
+        title: "Explore ways to earn",
+        description: "Work and earn in different ways. Bid for jobs. Pitch your projects. Discuss your in-demand skills with our recruiters so they can find opportunities aligned with your passions and career goals. Do all three. Create a predictable pipeline and build your network.",
+        imgSource: "/img/ResponsiveImage.png"
+    },
+    {
+        title: "Get paid securely",
+        description: "Choose how you get paid. Our fixed-price protection releases client payments at project milestones. Hourly protection bills clients every week. However you work, our service fees are the same. Spend less time chasing, more earning.",
+        imgSource: "/img/GetPaid.png"
+    }
+];
+
+const Container = styled.div`
+    display: flex; 
+    flex-direction: column; 
+    flex-shrink:0;
+    gap: 20px;
+    width:788px;
+    @media screen and (max-width:600px){
+        width:377px;
+    }
+`;
+
+const BodyContent = styled.div`
+    display: flex; 
+    align-items: center; 
+    gap: 22px;
+    @media screen and (max-width: 600px){
+        flex-wrap: wrap;
+        display: flex;
+        flex-direction: column;
+        align-item: flex-start;
+        width: 366px;
+    }
+`;
+
+const TextContainer = styled.div`
+    display: flex; 
+    flex-direction: column; 
+    @media screen and (max-width: 600px){
+        flex-direction: row;
+        color: red;
+    }
+`;
+
+const SmallScreenContainer = styled.div`
+display: none;
+justify-content: space-between;
+align-items:items;
+gap: 20px;
+@media screen and (max-width: 600px){
+        display: flex;
+    }
+`;
+
+const VXCont = styled.div`
+    display: none;
+    @media screen and (max-width: 600px){
+        display: block;
+    }
+    `;
+
+const ImageContainerResponsive = styled.div`
+    width: 127px; 
+    height: 100px;
+    border-radius: 10px;
+
+    @media screen and (max-width: 600px){
+        display: block;
+    }
+`
+
+const HowItWorksContentWrapper = () => {
+    const isSmallWindow = useWindowWidthEventListener(600);
+    const [renderCards, setRenderCards] = useState([]);
+
+    useEffect(() => {
+        isSmallWindow ? setRenderCards(labelsSMArray) : setRenderCards(labelsArray);
+    }, [isSmallWindow]);
+
+    return (
+        <>
+            <Wrapper>
+                <Container>
```

Comment:
What is the purpose of this container and wrapper here? Are both needed, and should margin be added here?